draft-ietf-gnap-core-protocol.md

```diff
@@ -1813,8 +1813,8 @@ an opaque identifier.
 }
 ~~~
 
-In this example, the AS is returning only a pair of [subject identifiers](#response-subject)
-as both an email address and an opaque identifier.
+In this example, the AS is returning only a set of [subject identifiers](#response-subject),
```

Comment:
I would insert a comma in front of "and a decentralized identifier (DID)".

Reply:
```suggestion
In this example, the AS is returning set of [subject identifiers](#response-subject),
```